apis/python/src/tiledbsoma/_indexer.py

```diff
@@ -29,15 +29,15 @@
 def tiledbsoma_build_index(
     data: IndexerDataType, *, context: Optional["SOMATileDBContext"] = None
 ) -> IndexLike:
-    """Initialize re-indexer for provided indices.
+    """Initialize re-indexer for provided indices (deprecated).
```

Comment:
I found that sphinx really likes to take _only_ the first sentence and use it in various places for summaries. The deprecation status of this is important, and it belongs in that first sentence.